src/main/java/com/b44t/messenger/DcContext.java

```diff
@@ -211,6 +213,14 @@ protected void finalize() throws Throwable {
     public native boolean      isSendingLocationsToChat(int chat_id);
     public DcProvider          getProviderFromEmailWithDns (String email) { long cptr = getProviderFromEmailWithDnsCPtr(email); return cptr!=0 ? new DcProvider(cptr) : null; }
 
+    public boolean isMentionsEnabled() {
+      return !"0".equals(getConfig(CONFIG_MENTION_NOTIF_ENABLED));
```

Comment:
while working, it is a bit hard to understand, every string by "0" means "enabled" then, so also "" (which is the default). getConfigInt() would not work as well, although used for setting.

suggestion: call the thing `CONFIG_NOTIIFY_MENTIONS_IF_MUTED` / `ui.notify_mentions_if_muted`, then the default of "" or 0 matches and we can use get/setConfigInt(), and that is also in sync with the other options

Reply:
> suggestion: call the thing `CONFIG_NOTIIFY_MENTIONS_IF_MUTED` / `ui.notify_mentions_if_muted`, then the default of "" or 0 matches and we can use get/setConfigInt(), and that is also in sync with the other options

renaming would be ok, but I don't see how it changes the problem, "" (unset) still needs to mean "1" (enabled) since it is the default, you want to "ui.notify_mentions_if_muted" to be on by default :thinking:

Reply:
I renamed it :+1: and I think I improved the code to be easier to understand now

Reply:
sorry, my fault 🤦‍♂️ 

i meant to inverse the meaning of the var, so `CONFIG_MUTE_MENTIONS_IF_MUTED`; that way our default matches the default core will return and we do not need to deal with that.

in fact, we had the default-passing in old cores, but got rid of that as it turned out to be tricky and error-prone in detail. tbh, i would prefer to not introduce a getConfig(key, default) again, at least not for a comparable side-quested thing

Reply:
i also think, it just does not work. getConfig("ui.unkown_sth") returns an empty string, not null.

the try/catch is only to catch Integer-parsing exceptions

Reply:
I renamed it again